easybuild/easyconfigs/b/bokeh/bokeh-3.2.1-foss-2022b.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PythonBundle'
+
+name = 'bokeh'
+version = '3.2.1'
+
+homepage = 'https://github.com/bokeh/bokeh'
+description = "Statistical and novel interactive HTML plots for Python"
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+builddependencies = [
+    ('setuptools', '64.0.3'),
+    ('meson-python', '0.11.0'),
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('matplotlib', '3.7.0'),
+    ('PyYAML', '6.0'),
+    ('Pillow', '9.4.0'),
+    ('SciPy-bundle', '2023.02'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('tornado', '6.3.2', {
+        'checksums': ['4b927c4f19b71e627b13f3db2324e4ae660527143f9e1f2e2fb404f3a187e2ba'],
+    }),
+    ('contourpy', '1.0.7', {  # stick to 1.0.7 to avoid need for newer Meson / meson-python
```

Comment:
I'm fine with sticking with this version of contourpy, but since Meson/meson-python are only build dependencies, I guess having multiple versions in the same toolchain is not necessarily problematic, right? Anyway, up to you if you want to keep it this way :)